sdks/python/apache_beam/runners/portability/fn_api_runner.py

```diff
@@ -139,6 +139,25 @@ def done(self):
       self._state = self.DONE_STATE
 
 
+class _PartitionBuffer(object):
+  """This class is created to support partition(n) function."""
+  def __init__(self, inputs):
+    self._inputs = inputs
+
+  def partition(self, n):
+    v = list(self._inputs.values())[0]
```

Comment:
The check, if any, should be inside the loop and per input. This code (implicitly) assumes exactly one element in inputs--we shouldn't do so without explicitly asserting that. Better would be to handle general dicts.

Reply:
This is removed now.